src/config_terrain.c

```diff
@@ -490,6 +514,245 @@ TbBool parse_terrain_slab_blocks(char *buf, long len, const char *config_textnam
+                    COMMAND_TEXT(cmd_num),block_buf,config_textname);
+            }
+            break;
+            case 4:
+            if (get_conf_parameter_single(buf,&pos,len,word_buf,sizeof(word_buf)) > 0)
+            {
+                k = atoi(word_buf);
+                if (k >= 0)
+                {
+                    slbattr->block_health_index = k;
+                    n++;
+                }
+            }
+            if (n < 1)
+            {
+                CONFWRNLOG("Incorrect value of \"%s\" parameter in [%s] block of %s file.",
+                    COMMAND_TEXT(cmd_num),block_buf,config_textname);
+            }
+            break;
+            case 5:
+            case 6:
+            {
+                unsigned long *flg = (cmd_num == 5) ? &slbattr->block_flags : &slbattr->noblck_flags;
+                *flg = 0;
+                while (get_conf_parameter_single(buf,&pos,len,word_buf,sizeof(word_buf)) > 0)
+                {
+                    k = get_id(terrain_flags, word_buf);
+                    switch(k)
+                    {
+                        case 1:
+                        {
+                            *flg |= SlbAtFlg_Valuable;
+                            break;
+                        }
+                        case 2:
+                        {
+                            *flg |= SlbAtFlg_IsRoom;
+                            break;    
+                        }
+                        case 3:
+                        {
+                            *flg |= SlbAtFlg_Unexplored;
+                            break;    
+                        }
+                        case 4:
+                        {
+                            *flg |= SlbAtFlg_Digable;
+                            break;   
+                        }
+                        case 5:
+                        {
+                            *flg |= SlbAtFlg_Blocking;
+                            break;    
+                        }
+                        case 6:
+                        {
+                            *flg |= SlbAtFlg_Filled;
+                            break;    
+                        }
+                        case 7:
+                        {
+                            *flg |= SlbAtFlg_IsDoor;
+                            break;    
+                        }
+                        case 8:
+                        {
+                            *flg |= SlbAtFlg_TaggedValuable;
+                            break;   
+                        }
+                        default:
+                        {
+                            CONFWRNLOG("Incorrect value of \"%s\" parameter \"%s\" in [%s] block of %s file.",
+                            COMMAND_TEXT(cmd_num),word_buf,block_buf,config_textname);
+                            break;
+                        }
+                    }
+                    n++;
+                }
+                break;
+            }
+            /*
```

Comment:
Isn't that the code to actually load the values?

Reply:
But you commented it out,...

Reply:
See above. BlockFlags and NoBlockFlags are merged into one case statement.

Reply:
Well, if you remove code, delete it. No need to keep it there as comments.

Reply:
What I mean by that, please delete the code when you believe the pull is ready to be merged otherwise. I won't merge pulls where there is still stuff remaining that needs cleaning up.